### DISCUSSION TITLE
[Question] Necessary procedure to import URDF files

### DISCUSSION THREAD

**Discussion by catachiii** [isaac-sim/IsaacLab#545] [Q&A]
### Description

Dear IsaacLab team. Thanks for providing us with up-to-date USD files for all kinds of humanoids. I have couple of questions about the necessary procedures to import the URDFs by myself. 

Taking the URDFs of Unitree H1 for example (https://github.com/unitreerobotics/unitree_ros/tree/master/robots/h1_description/urdf), I have been trying to convert the given URDF to USD files with the How-to guide (https://isaac-sim.github.io/IsaacLab/source/how-to/import_new_asset.html). 

I successfully converted the URDF with `--merge-joints` and `--make-instanceable`. However, when I'm loading the humanoid USD file similarly to the demos (https://github.com/isaac-sim/IsaacLab/blob/main/source/standalone/demos/bipeds.py). I encountered the issue that some of the visuals (like the logos, etc.) are not updated with physics, as shown below. 

![image](https://github.com/isaac-sim/IsaacLab/assets/33168198/28a26405-9d7a-4f11-a57b-3d1b13b3fd78)

I'm not sure what I missed when converting. So far, all quadruped URDFs from Unitree work fine and similarly to what is provided from the Nucleus server. 

Thanks for your help!

Jin

PS:
I also noticed there was a PR weeks ago importing Unitree G1 successfully. @abizovnuralem @Dhoeller19 would you mind sharing your experience with me?

**Comment by fooyuhui** [comment]
hello, can you try to use this USD file?
[h1_with_hand.zip](https://github.com/user-attachments/files/15911814/h1_with_hand.zip)
I tested it, and it can work well. This way you can tell if there's a problem with the file or with the code.

**Comment by catachiii** [comment]
@fuyh20 It looks good to me although I think it is wrongly converted with fixed base, as the robot remains in the air. 

From my side I think I didn't modify any code. I was just replacing the usd files and it seems not to work. That's why I was puzzled. 

![image](https://github.com/isaac-sim/IsaacLab/assets/33168198/88959a73-4fd2-4e24-b7db-e4e9c4ef38e0)

**Comment by catachiii** [comment]
Update:

I tried without `--merge_joints`, and now the visuals look all good. But the simulation can easily explode with the following error

`2024-06-20 11:01:56 [57,157ms] [Error] [omni.physx.plugin] PhysX error: Illegal BroadPhaseUpdateData 
, FILE /builds/omniverse/physics/physx/source/lowlevelaabb/src/BpBroadPhaseABP.cpp, LINE 4023`

![image](https://github.com/isaac-sim/IsaacLab/assets/33168198/54936a77-2c55-4bc7-8bb7-d6286bf28786)

**Reply by Lr-2002** [reply]
hi, have you solve the problem with illegal broadphase update data?
